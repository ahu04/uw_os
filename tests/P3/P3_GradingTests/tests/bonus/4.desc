Job control and ampersand.  Score: 3