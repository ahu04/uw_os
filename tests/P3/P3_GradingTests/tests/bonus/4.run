sleep 10.0 | sleep 10.1 | sleep 10.2 &
##sleep 0.2
##assertStat S sleep 10.0
##assertStat S sleep 10.1
##assertStat S sleep 10.2
\x03
##assertStat S sleep 10.0
##assertStat S sleep 10.1
##assertStat S sleep 10.2
jobs
fg 1
##sleep 0.2
\x03
##sleep 0.2
##assertStat X sleep 10.0
##assertStat X sleep 10.1
##assertStat X sleep 10.2
exit
