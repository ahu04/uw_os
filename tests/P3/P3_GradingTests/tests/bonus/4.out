${PROMPT}${PROMPT}1: sleep 10.0 | sleep 10.1 | sleep 10.2 &
${PROMPT}${PROMPT}